.gitignore

```diff
@@ -1,6 +1,13 @@
 .DS_Store
 /tmp
 
+# Ruby / Bundler
+.bundle/
```

Comment:
Running `bundle install --with development` created this. Ignoring it [seems to be the best practice](https://devcenter.heroku.com/articles/bundler-configuration) and it's what we've been doing [in other repos too](https://github.com/Automattic/simplenote-ios/blob/1bf42d514cab4aa127d8cbb70f25f5fef0a34d77/.gitignore#L38).